pubspec.yaml

```diff
@@ -78,6 +78,22 @@ flutter:
     - assets/images/signup/
     - assets/images/group/
     - assets/images/profile/
+    - android/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png
```

Comment:
Are you sure that these android/* assets are specifically required to be included ? I don't think so